cmds/cmd.c

```diff
@@ -196,7 +197,15 @@ void cmd_prompt(void)
 
 		/* Regular characters */
 		if (c) {
-			if (c == '\r') {
+			if ((c == '\r') || (c == '\n')) {
+				if (newline) {
```

Comment:
```suggestion
				if (newline != 0) {
```

Reply:
You are right, sorry for that. The last force-push handles this case.